custom-login/README.md

```diff
@@ -26,7 +26,8 @@ Plug these values into the `mvn` commands used to start the application.
 cd custom-login
 mvn -Dokta.oauth2.issuer=https://{yourOktaDomain}/oauth2/default \
     -Dokta.oauth2.clientId={clientId} \
-    -Dokta.oauth2.clientSecret={clientSecret}
+    -Dokta.oauth2.clientSecret={clientSecret} \
+    -Dokta.oauth2.postLogoutRedirectUri={postLogoutRedirectUri}
```

Comment:
We probably don't want to add this one here, but if we do add it, we would need to include details on how to configure it.
My suggestion is to move this out to a different pr (if it needs to be added)

Reply:
Addressed this in next commit.